### ISSUE TITLE
No .cmt files in opam installed eliom, no merlin

### ISSUE THREAD

**Issue by aryx** [ocsigen/eliom#649]
Most OCaml libraries comes with a set of .cmt files in ~/.opam/.../lib/<package>/
so tools like merlin can help programmers find the definition of a function.

Is there any reason eliom does not follow the usual conventions and installed .mli and .cmt in
the opam installed directory?

**Comment by aryx** [comment]
Is this project still alive? Where should I direct my feature request?

**Comment by aryx** [comment]
I'd love to use Eliom, I really like the idea of one language for both the server and client and seamless communication between them, but if the project is not active anymore, I'll switch to something else.

**Comment by balat** [comment]
Yes it is very active. We are using it every day at Be Sport. Just a little bit busy sorry.

**Comment by aryx** [comment]
Actually the installed opam package of eliom should contain all the .cmt and the .mli for merlin to be able to jump to definition easily. I suggest to just follow what 'dune install' does for any library.

**Comment by balat** [comment]
Thank you for the suggestion. It is very welcome! Feel free to open a pull request if you want to contribute. Ocsigen is a collaborative project.
